scaleway/resource_iot_device.go

```diff
@@ -319,11 +335,32 @@ func resourceScalewayIotDeviceRead(ctx context.Context, d *schema.ResourceData,
 		_ = d.Set("message_filters", []map[string]interface{}{mf})
 	}
 
+	////
+	// Read Device certificate
+	////
+
+	// As we cannot read the key, we get back it from cache and do not change it.
```

Comment:
@grom-42 I can't find any code related with your comment. Can you give an inside?